pkg/doc/presexch/definition_test.go

```diff
@@ -39,6 +41,286 @@ func TestPresentationDefinition_IsValid(t *testing.T) {
 	})
 }
 
+func TestPresentationDefinition_CreateVP(t *testing.T) {
```

Comment:
We should include tests that check whether the output VP is formed correctly (should pass a parsing check). This is related to by comment [here](https://github.com/hyperledger/aries-framework-go/pull/2407/files#r545164335).

Reply:
I will add this check

Reply:
@llorllale added `context` and `type`. Also added check in tests whether the VP is parsable.